assets/js/params/Custom/Code/AceEditor.js

```diff
@@ -74,15 +75,15 @@ export default class WorkbenchAceEditor extends React.PureComponent {
     return (
       <>
         <div className='help'>
-          Define a <kbd>process(table)</kbd> function that accepts
-          a <kbd>pd.DataFrame</kbd> and returns
-          a <kbd>pd.DataFrame</kbd>. You may use
-          the <a target='_blank' rel='noopener noreferrer' href='https://docs.python.org/3/library/math.html'><kbd>math</kbd></a>,
-          {' '}<kbd>pd</kbd>{' '}
-          (<a target='_blank' rel='noopener noreferrer' href='https://pandas.pydata.org/pandas-docs/stable/api.html#dataframe'>Pandas</a>)
-          {' '}and <kbd>np</kbd>{' '}
-          (<a target='_blank' rel='noopener noreferrer' href='https://docs.scipy.org/doc/numpy/reference/routines.html'>Numpy</a>)
-          {' '}modules.
+          <Trans id='js.params.Custom.Code.AceEditor.help' description='The tags <3>, <6>, and <8> are URLs. The rest are code formatting. Please keep code and names of libraries untranslated.'>
+              Define a <kbd>process(table)</kbd> function that accepts
+              a <kbd>pd.DataFrame</kbd> and returns
+              a <kbd>pd.DataFrame</kbd>. You may use
+              the <a target='_blank' rel='noopener noreferrer' href='https://docs.python.org/3/library/math.html'><kbd>math</kbd></a>, <kbd>pd</kbd>{' '}
+              (<a target='_blank' rel='noopener noreferrer' href='https://pandas.pydata.org/pandas-docs/stable/api.html#dataframe'>Pandas</a>) {' '}
```

Comment:
When you put `{' '}` after a _space_, I think that produces the wrong HTML: it becomes `)  and` instead of `) and` (there's an extra space). Please don't do that.

That said, I'll merge because I hope CSS `white-space: normal;` will hide the error.

Reply:
This is parsed by lingui and stored in the translation file for English, from where it is fetched to show to the user. The message in the catalog has no double space, so Lingui parses this as we intend.